cogs/tags/tags.py

```diff
@@ -200,13 +205,14 @@ async def user_exceeds_tag_limit(self, server: discord.Guild, user: discord.Memb
 
         Returns:
         --------
-        bool
-            True if too many tags, else False.
+        (bool, int)
+            bool: True if user has too many tags, else False.
+            int: The maximum number of tags this user can have. -1 if they
```

Comment:
Returning infinity if the user has no limit would allow for simpler logic

Reply:
I'll look into using `float('inf')`.

Reply:
Used `float("inf")` to replace `NO_LIMIT`.